src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/entity/MixinEntityPlayerSP.java

```diff
@@ -365,6 +372,9 @@ public void onLivingUpdate() {
         if ((scaffold.getState() && !scaffold.getSprint()) || (sprint.getState() && !legitSprint && sprint.getCheckServerSide() && (onGround || !sprint.getCheckServerSideGround()) && !sprint.getAllDirections() && shouldStop))
             setSprinting(false);
 
+        if ((scaffold.getState() && scaffold.getSprint() && scaffold.getEagleSprint() && scaffold.getEagle() == "Normal" && mc.gameSettings.keyBindSprint.isKeyDown() && mc.gameSettings.keyBindSneak.isKeyDown()))
+            setSprinting(true);
```

Comment:
Do not use == for Strings in Java. This is NOT Kotlin code! Use .equals() or .equalsIgnoreCase().